common/tools/dev-tool/src/templates/sampleReadme.md.ts

```diff
@@ -214,10 +214,8 @@ ${fence(
   "bash",
   `node ${(() => {
     const firstSource = filterModules(info)[0].relativeSourcePath;
-    const fileName = info.useTypeScript
-      ? "dist/" + firstSource
-      : firstSource.replace(/\.ts$/, ".js");
-    return fileName;
+    const filePath = info.useTypeScript ? "dist/" : "";
```

Comment:
I wanted to write a test but I didn't see any tests, if I missed them please let me know and I'll be happy to add them